examples/meander/meander_hydro.py

```diff
@@ -0,0 +1,129 @@
+"""
+Meander Test case
+=======================
+
+Solves the initial hydrodynamics simulation of a meander.
+
+Note this is not the main run-file and is just used to create an initial checkpoint for
+the morphodynamic simulation.
+
+For more details of the test case set-up see
+[1] Clare et al. 2020. “Hydro-morphodynamics 2D Modelling Using a Discontinuous
+    Galerkin Discretisation.” EarthArXiv. January 9. doi:10.31223/osf.io/tpqvy.
+
+"""
+
+from thetis import *
+
+import numpy as np
+import time
+
+# define mesh
+mesh2d = Mesh("meander.msh")
+x, y = SpatialCoordinate(mesh2d)
+
+# define function spaces
+V = FunctionSpace(mesh2d, 'CG', 1)
+P1_2d = FunctionSpace(mesh2d, 'DG', 1)
+vectorP1_2d = VectorFunctionSpace(mesh2d, 'DG', 1)
+
+# define underlying bathymetry
+
+bathymetry_2d = Function(V, name='Bathymetry')
+
+gradient = Constant(0.0035)
+
+L_function = Function(V).interpolate(conditional(x > 5, pi*4*((pi/2)-acos((x-5)/(sqrt((x-5)**2+(y-2.5)**2))))/pi, pi*4*((pi/2)-acos((-x+5)/(sqrt((x-5)**2+(y-2.5)**2))))/pi))
+
+bathymetry_2d1 = Function(V).interpolate(conditional(y > 2.5, conditional(x < 5, (L_function*gradient) + 9.97072, -(L_function*gradient) + 9.97072), 9.97072))
```

Comment:
Are these the same fields as in the main file? Then would be better to define all shared fields/expressions in a separate file like `meander_setup.py` and import here

Reply:
I have added a meander_setup file